ui/hooks/useMultichainTransactionDisplay.ts

```diff
@@ -40,7 +40,7 @@ export function useMultichainTransactionDisplay(
   networkConfig: MultichainProviderConfig,
 ) {
   const locale = useSelector(getIntlLocale);
-  const isNegative = transaction.type === TransactionType.Send;
+  const isNegative = transaction.type !== TransactionType.Receive;
 
   const assetInputs = aggregateAmount(
```

Comment:
Might be a bit better so if one day we use the `to` for Swaps it will be positive. 

```suggestion
    const assetInputs = aggregateAmount(
    transaction.from as Movement[],
    true,
    locale,
    networkConfig.decimals,
  );
  const assetOutputs = aggregateAmount(
    transaction.to as Movement[],
    transaction.type === TransactionType.Send,
    locale,
    networkConfig.decimals,
  );
  const baseFee = aggregateAmount(
    transaction.fees.filter((fee) => fee.type === 'base') as Movement[],
    true,
    locale,
  );
  const priorityFee = aggregateAmount(
    transaction.fees.filter((fee) => fee.type === 'priority') as Movement[],
    true,
    locale,
  );
```